### DISCUSSION TITLE
Rename Autoloads to Global Scripts

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#10021] [General Discussions]
### **PROPOSAL**: Rename Autoloads to Global Scripts.

Autoloads - What is it? Automatic loading? of what? The naming is meaningless = bad naming.

Rule 1 - give everything excellent names.

It's actually Global Scripts and should be named that.

"The choice of a name should be mnemonic — that is, designed to indicate to the casual observer the intent of its use" - Founder

**Comment by theraot** [comment]
An autoload can be an scene, so global scripts would NOT be accurate.

---

Meanwhile... The project settings changed recently, it looks like this now (4.3 beta 2):

![image](https://github.com/godotengine/godot-proposals/assets/1212509/d718ff48-4768-46ae-b065-cf456168d7b9)

And there seems an agreement to minimize repeating the word "Global" (Global Groups was renamed to Groups).

**Reply by shak2** [reply]
Perfect! Case closed.
